src/llamacrawl/services/readers/firecrawl.py

```
@@ -917,32 +926,30 @@ async def _async_crawl_with_polling(
 
             await asyncio.sleep(poll_interval)
 
         if status == "failed":
-            raise RuntimeError(
-                f"Firecrawl crawl job {job_id} failed to complete successfully"
-            )
+            raise RuntimeError(f"Firecrawl crawl job {job_id} failed to complete successfully")
 
+        doc_types = [type(d).__name__ for d in collected_docs[:3]] if collected_docs else []
         self.logger.debug(
             f"Polling crawl completed with {len(collected_docs)} total documents",
```

Ruff (0.14.0)

Logging statement uses f-string

(G004)